expression/builtin_time_test.go

```diff
@@ -2404,8 +2404,6 @@ func (s *testEvaluatorSuite) TestTimeFormat(c *C) {
 		Input  []string
 		Expect interface{}
 	}{
-		{[]string{"100:00:00", `%H %k %h %I %l`},
```

Comment:
Why delete this test case?

Reply:
`Time` will not represent Time of Mysql, hour will not greater than 24.

Reply:
And we only have 5 bit for hour